.circleci/config.yml

```diff
@@ -40,7 +40,7 @@ jobs:
     machine:
       image: ubuntu-2004:202010-01
     environment:
-      KYVERNO_VERSION: v1.5.0-rc3
+      KYVERNO_VERSION: v1.5.7
```

Comment:
This was necessary to make the `Validate policies` step work. With v1.5.0-rc3 the step failed with this error:

```
Policy restrict-cluster-label-service-priority is invalid.
Error: invalid policy.
Cause: path: spec.rules[0].match.resources.match: values[0][giantswarm.io/service-priority]: Invalid value: "*": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue',  or 'my_value',  or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```

Patterns have been introduced with Kyverno 1.5.0.